validator/rpc/server.go

```diff
@@ -170,50 +169,50 @@ func (s *Server) InitializeRoutes() error {
 		return errors.New("no router found on server")
 	}
 	// Adding Auth Interceptor for the routes below
-	s.router.Use(s.AuthTokenHandler)
+	s.AuthTokenHandler(s.router)
```

Comment:
this is a middleware, please include this ... ( I wonder if something will break on the middleware registration because of this though , it needs to ignore some of the paths)